lsp-dart.el

```diff
@@ -140,7 +140,7 @@ Defaults to side following treemacs default."
 
 (defun lsp-dart--assert-sdk-min-version (version)
   "Assert dart sdk min version is VERSION."
-  (cl-assert (string-prefix-p version (lsp-dart--get-dart-version))
+  (cl-assert (version<= version (lsp-dart--get-dart-version))
```

Comment:
Sadly, this comparison should not work for the dev channel: `2.8.9-dev...` :/

Reply:
I changed to `string<` that compare string and It seems to resolve our problem :)